const-str/src/lib.rs

```diff
@@ -74,7 +75,11 @@ pub mod __imp {
 
     #[cfg(feature = "case")]
     pub use const_str_proc_macro::{
-        to_camel_case, to_kebab_case, to_shouty_kebab_case, to_shouty_snake_case, to_snake_case,
+        to_camel_case,
+        to_kebab_case,
+        to_shouty_kebab_case,
+        to_shouty_snake_case,
+        to_snake_case,
     };
 }
```

Comment:
Just run `cargo format` to make CI happy
<https://github.com/Nugine/const-str/pull/3/checks?check_run_id=2043751881>

Reply:
Figured out why this wasn't happening -- `cargo fmt` was using my config file instead of the defaults. Might be worth plopping a blank `rustfmt.toml` in the project root to override personal configs in the future.